### ISSUE TITLE
Create ROS node for RS

### ISSUE THREAD

**Issue by JacksonStillwagon** [UA-NASA-Robotics/2024-2025-Robot-Code#4]
Once the file struct is decided and a package is made we have to begin filling that package. We'll start will the RS (remote station node). The objective of this node is as follows:

1. Receive inputs (likely joystick values, and buttons) from a human
2. Publish to a topic those values
3. Display sensor data (this will happen *much* later down the line)

Again **THIS CAN NOT BE DONE UNTIL** the Design File Structure issue is completed

**Comment by JacksonStillwagon** [comment]
Look back on the Design File Structure for details

I plan to make the package on 9/19 or 9/26 as a demonstration of ROS. It will still need filled with actual code, so we'll start getting that together

**Comment by JacksonStillwagon** [comment]
Using the ROS Diagram we decided on, we've split our architecture into 4 different packages. I have made those packages as C++ packages. That is because it is easier to add python to C++ packages then the other way around, and we are now doing a mixed language base. Instruction on mixing the packages are here: [C++ and Python Nodes](https://www.theconstruct.ai/how-to-use-c-and-python-in-the-same-ros2-package_ros_english_tutorial/)
